packages/firestore/package.json

```diff
@@ -26,6 +26,7 @@
     "prepare": "yarn build"
   },
   "main": "dist/index.node.cjs.js",
+  "mainES2017": "dist/index.node.esm2017.js",
```

Comment:
`main-esm2017` to be consistent with the existing convention? We have a `lite-esm2017` target in @firebase/app which is used to produce `firebase-performance-standalone.es2017.js`